replication/gatys_ecker_bethge_2016/main.py

```diff
@@ -79,31 +80,29 @@ def figure_3(args):
     for layers, score_weight in itertools.product(layer_configs, score_weights):
         row_label = layers[-1]
         column_label = f"{1.0 / score_weight:.0e}"
-        header = (
+        print(
             f"Replicating Figure 3 image in row {row_label} and column {column_label}"
         )
-        with args.logger.environment(header):
-            hyper_parameters.style_loss.layers = style_layers
-            if args.impl_params:
-                hyper_parameters.style_loss.layer_weights = paper.compute_layer_weights(
-                    style_layers
-                )
-            hyper_parameters.style_loss.score_weight = score_weight
-
-            output_image = paper.nst(
-                content_image,
-                style_image,
-                impl_params=args.impl_params,
-                hyper_parameters=hyper_parameters,
-                quiet=args.quiet,
-                logger=args.logger,
+        hyper_parameters.style_loss.layers = layers
```

Comment:
Using `layers` here is actually correct, since `style_layers` is constant and we want some change through the loop. I'm ok with keeping the buggy behavior here if you open an issue or send a PR so we don't forget.